src/Select/Select/presets/0-default.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Select from '../Select';
+import SelectItem from '../../SelectItem/SelectItem';
```

Comment:
Please sort imports by modules.
If you use PhpStorm you could make IDE do it for you here:
![image](https://user-images.githubusercontent.com/3602930/51845371-bcc8b180-2317-11e9-9b7a-c5e0d843919b.png)